spec/openapi.yaml

```diff
@@ -1697,6 +1697,51 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/404response'
+  /v1/messaging/numbers/sender_address/requests/{id}/reverify:
+    post:
+      security:
+        - basic_auth: [ ]
+        - hmac_auth: [ ]
+      tags:
+        - Source Address
+      summary: Re-verify Sender Address
+      operationId: ReverifySenderAddressCodePost
+      parameters:
+        - name: id
+          in: path
+          description: '36 character UUID.'
+          required: true
+          style: simple
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Created
```

Comment:
`Sender address request is created for re-verification`?
Can you please ask Jack a better description for this?